clang/include/clang/Support/RISCVVIntrinsicUtils.h

```diff
@@ -485,7 +485,7 @@ class RVVIntrinsic {
 
 // RVVRequire should be sync'ed with target features, but only
 // required features used in riscv_vector.td.
-enum RVVRequire : uint16_t {
+enum RVVRequire : unsigned int {
```

Comment:
Use uint32_t since we're counting bits.

Reply:
Done.  Note, I also changed the type in `RVVIntrinsicRecord::RequiredExtensions` and `SemaRecord::RequiredExtensions`.